openpype/tools/utils/models.py

```diff
@@ -330,6 +330,19 @@ def refresh(self):
         if new_items:
             root_item.appendRows(new_items)
 
+    def get_index(self, project_name):
```

Comment:
I think find would be better in this case.
```suggestion
    def find_project(self, project_name):
```